api/src/utils/passwordStrengthCheck.ts

```diff
@@ -0,0 +1,19 @@
+import zxcvbn from 'zxcvbn';
+
+/**
+ * Checks if the given password is strong enough
+ * Returns `true` if the password passes the strength test
+ */
+
+export function passwordStrengthCheck(password: string): boolean {
+	const result = zxcvbn(password);
+	const guessable = result.score < 3;
+	const minLength = password.length < 15;
+	const nonUnicodeCharFound = password.match(/\p{Cc}/iug);
+
+	if (guessable || minLength || nonUnicodeCharFound) {
+		return false;
+	}
```

Comment:
```suggestion
	const hasControlCharacter = /\p{Cc}/iug.test(password);

	if (guessable || minLength || hasControlCharacter) {
		return false;
	}
```

Two nitpicks:
- Prefer `RegExp.test` for boolean checks as it just checks if the regex matches or not and don't create the extra things that `string.match` creates... Confusing, I know...
- Rename the variable to better represent it is a boolean.